tests/io/vasp/test_outputs.py

```diff
@@ -1558,6 +1558,20 @@ def test_as_dict_and_from_dict(self):
         )
 
 
+class TestAECCARS(PymatgenTest):
+    def test_write_file(self):
+        aeccar0_test = Chgcar.from_file(f"{TEST_FILES_DIR}/bader/AECCAR0.gz")
+        aeccar0_test.write_file("AECCAR0_test")
+        aeccar0_read = Chgcar.from_file("AECCAR0_test")
+        np.allclose(aeccar0_test.data["total"], aeccar0_read.data["total"])
```

Comment:
This needs to be `assert`ed. Just calling `np.allclose` doesn't fail the test if the check fails.

Reply:
Oh!!! Thanks for the good catch. I missed that.